rllib/policy/policy.py

```diff
@@ -171,6 +175,79 @@ class Policy(metaclass=ABCMeta):
     `rllib.policy.tf_policy_template::build_tf_policy_class` (TF).
     """
 
+    @staticmethod
```

Comment:
Analogous to Algorithm.from_checkpoint and Algorithm.from_state, but for individual polciies.